modules/tasks.js

```diff
@@ -0,0 +1,52 @@
+import Store from "./store";
+
+export default class AddDelete {
+    static displayTasks = () => {
+        const taskList = Store.getLocalStorage();
+        taskList.forEach((task) => AddDelete.addTask(task));
+    }
+
+    static addTask = (task) => {
+        const list = document.querySelector('.tasked-list');
+        const row = document.createElement('div');
+        row.classList.add('task-row');
+        row.innerHTML = `
+          <div class="task-element">
+            <div class="task-description">
+              <input type="checkbox" class="task-complete"/> 
+              <input type="text" class="task-text" value="${task.description}" />
+            </div> 
+            <div class="task-icon"><i class="fa-solid fa-ellipsis-vertical delete"></i></div>
+          </div>
+        `;
+
+        list.appendChild(row);
+    }
+
+    static editTask = () => {
+      const tasksList = Store.getLocalStorage();
+      const editField = document.querySelectorAll('.task-text');
+      editField.forEach((field, index) => {
+        tasksList[index].description = field.value;
+      })
+
+      localStorage.setItem('tasks', JSON.stringify(tasksList));
+    }
```

Comment:
- [ ] **Good job on implementing most of the functions on your project 👍** However, some features are missing in your project 😎😎
    - The completed task is not being updated in the local storage, I think it would be better to update the local storage on every change made by the user on the completed task 👍 👍 
![image](https://user-images.githubusercontent.com/102757126/213441605-7939ea92-4b0d-45cb-adc0-234ffa3096ef.png)

    - It would be better also to provide an option of deleting a selected item on each task as suggested in the project requirements 🙏 🙏 


### KEEP IT UP 👍

Reply:
What do you mean option of deleting a selected item? The button does that

Reply:
Hey thank you for your feedback. Kindly note that this is the second milestone yet you are suggesting changes for the third milestone. I have implemented it anyway. Thank you once again